### PR TITLE
(pC-158) routes for music and show types

### PR DESCRIPTION
**Pull request by Ledoux**

### CODE REVIEW

File: routes/music_types.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from flask import current_app as app, jsonify
+from flask_login import current_user, login_required
```
Comment (atusithi) [inline routes/music_types.py]: import inutile de current_user

File: routes/show_types.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from flask import current_app as app, jsonify
+from flask_login import current_user, login_required
```
Comment (atusithi) [inline routes/show_types.py]: import inutile de current_user

Comment (Ledoux) [reply routes/show_types.py]: 👍

File: routes/music_types.py (hunk 1 above)
Comment (Ledoux) [reply routes/music_types.py]: 👍

**Pull request merged**